database/insertData.js

```diff
@@ -442,6 +453,57 @@ const queries = [
                           "Producer"
                         ]
                       }
+                    },
+                    {
+                      code: "S1PB1"
+                      index: 3
+                      title: "Page Break"
+                      elementTypePluginCode: "pageBreak"
+                      category: INFORMATION
+                      parameters: { previousValidityCheck: true }
+                    },
+                    {
+                      code: "S1T2"
+                      index: 4
+                      title: "Intro Section 1 - Page 2/2"
+                      elementTypePluginCode: "textInfo"
+                      category: INFORMATION
+                      parameters: { title: "Company nationality" }
+                    },
+                    {
+                      code: "S1Q3"
+                      index: 5
+                      title: "Organisation national or international"
+                      elementTypePluginCode: "radioChoice"
+                      category: QUESTION
+                      parameters: {
+                        label: "Select the nationality of this company:"
+                        options: [
+                          "National",
+                          "International"
+                        ]
+                      }
+                    },
+                    {
+                      code: "S1Q4"
+                      index: 6
+                      title: "Import permit upload"
+                      elementTypePluginCode: "textInfo"
```

Comment:
```suggesition
elementTypePluginCode: "textInfo"
category: QUESTION
```
Needs category -- otherwise doesn't show on page.

Reply:
Oh! You solved a mystery to me! 🌚